README.md

```diff
@@ -116,6 +116,21 @@ this larger namespace.  But, we don't want to be so general that the spec become
    specified by the RSS \<language\> element.  If the rel="captions" attribute is present, the linked file is considered to be a closed captions file, regardless of what the
    mime type is.  In that scenario, time codes are assumed to be present in the file in some capacity.
 
+- **\<podcast:chapters/>**
+
+   Channel (optional | single)
+
+   This element lists optional `<podcast:chapter/>` item elements.
+
+- **\<podcast:chapter start="[(Normal Play Time)]" title="[(string)]" href="[(uri to content)]" type="[mime type]"/>**
+
+   Item (optional | multiple)
+
+   This element specifies a point in time during the podcast that can be linked to directly and optionally supplemented with additional content.
+   `start` is required and uses the [Normal Play Time](https://www.w3.org/TR/media-frags/#naming-time) standard.
```

Comment:
From looking at that media-frag document, Is this really a `start` time or is it more like an `interval`? Are you expecting a format like "start_seconds, end_seconds"?

Reply:
Good question. I'm open to using an interval instead. My reasoning for only using is as follows:
- The Simple Chapter Spec only used Start Time so I figured it would be good to default to that unless an interval was needed (maybe it is)
-  From my experience implementing chapters in a client I simply generate an End Time for a chapter by subtracting 1 sec from the next chapter's start time. I think the advantage here is that you don't need to worry as a client developer that the RSS chapter end times are incorrect or inconsistent. If I can generate them myself then I know the chapters intervals are correct. 
- I think intervals could be represented with just start times by including a chapter for `Start Commercial` and a chapter for `End Commercial` if that use case did arise, but admittedly it's not very clean

These aren't arguments against using a time interval instead, just why I went with `Start Time`. I think intervals would be slightly more complicated to support and implement for clients, but would add some functionality some clients might want. So I'm definitely open to that if we agree that would be useful and worth the minor complexity trade-off.

Reply:
Perfect! I just went through our chapter data in Buzzsprout and we only store the start time. You may just want to say the `start` is the start time in seconds for the chapter marker. I think that w3.org article may be more confusing.